include/z64save.h

```diff
@@ -1095,8 +1099,12 @@ typedef enum {
 #define WEEKEVENTREG_64_01 PACK_WEEKEVENTREG_FLAG(64, 0x01)
 #define WEEKEVENTREG_64_02 PACK_WEEKEVENTREG_FLAG(64, 0x02)
 #define WEEKEVENTREG_64_04 PACK_WEEKEVENTREG_FLAG(64, 0x04)
-#define WEEKEVENTREG_64_08 PACK_WEEKEVENTREG_FLAG(64, 0x08)
-#define WEEKEVENTREG_64_10 PACK_WEEKEVENTREG_FLAG(64, 0x10)
+
+// Two-bit field storing player form when first talked to Tingle that cycle
+//  0 - Zora, 1 - Deku, 2 - Goron, 3 - Human
+#define WEEKEVENTREG_TINGLE_KNOWN_PLAYER_FORM_LOW_BIT PACK_WEEKEVENTREG_FLAG(64, 0x08)
+#define WEEKEVENTREG_TINGLE_KNOWN_PLAYER_FORM_HIGH_BIT PACK_WEEKEVENTREG_FLAG(64, 0x10)
```

Comment:
```suggestion
#define WEEKEVENTREG_TINGLE_RECOGNIZED_PLAYER_FORM_LOW_BIT PACK_WEEKEVENTREG_FLAG(64, 0x08)
#define WEEKEVENTREG_TINGLE_RECOGNIZED_PLAYER_FORM_HIGH_BIT PACK_WEEKEVENTREG_FLAG(64, 0x10)
```
I think this would go better with `EnBal_SetRecognizedPlayerForm`